test/unit/core.js

```diff
@@ -439,6 +439,8 @@ QUnit.test( "isFunction", function( assert ) {
 	fn = function() {};
 	assert.ok( jQuery.isFunction( fn ), "Normal Function" );
 
+	assert.ok( !jQuery.isFunction( Object.create( fn ) ), "custom Function subclass" );
```

Comment:
We can use `notOk` here instead? It seems more explicit that way, nitpick of course